steps/mentor-calibre-lvs/configure.yml

```diff
@@ -20,21 +20,23 @@ inputs:
 outputs:
   - lvs.report
   - design.schematic.spi
+  - design_merged.lvs.v
 
 #-------------------------------------------------------------------------
 # Commands
 #-------------------------------------------------------------------------
 
 commands:
-  - if [ -z "{lvs_hcells_file}" ]; then export lvs_use_hcells=0; else export lvs_use_hcells=1; fi
-  - if [ -z "{lvs_connect_names}" ]; then export lvs_connect_names_state=NONE; else export lvs_connect_names_state=SOME; fi
+  - if [ -z $lvs_hcells_file ]; then export lvs_use_hcells=0; else export lvs_use_hcells=1; fi
+  - if [ -z $lvs_connect_names ]; then export lvs_connect_names_state=NONE; else export lvs_connect_names_state=SOME; fi
```

Comment:
Does this work if there are spaces in $lvs_connect_names?

```
% export foo="a b c"
% echo $foo
a b c
% if [ -z $foo ]; then echo yes; else echo no; fi
-bash: [: too many arguments
no
```

Reply:
Probably needs to be `if [ -z "$foo" ]` ?

Reply:
good call; fixed